### ISSUE TITLE
6-2 验证数据集的准确率跟损失计算有问题

### ISSUE THREAD

**Issue by myhrbeu** [lyhue1991/eat_tensorflow2_in_30_days#91]
你好， 非常感谢提供6-2的训练流程，三，自定义训练循环中最后验证集准确率跟损失计算有问题，我刚接触tf2，所没找到是哪里的问题。示例中验证集的损失明显过大，准确率过低。为了验证我的想法，我把训练集同时当做了验证集，train_model(model,train_dataset,train_dataset,EPOCH)
Epoch=5,Loss:0.30140844,Accuracy:0.917645752,Valid Loss:3.73952508,Valid Accuracy:0.430102259 结果中差别过大。
如果方便的话麻烦调试一下，万分感谢！
